test/mlir/onnx/onnx_structure.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: onnx-mlir-opt %s -split-input-file | FileCheck %s
+// RUN: onnx-mlir-opt %s | FileCheck %s
```

Comment:
split-input still removed here (and maybe others)

Reply:
it was redundant in this case, there's no delimiter in this file.